### DISCUSSION TITLE
What model/setup will work with Raw Text file training?

### DISCUSSION THREAD

**Discussion by techogre** [oobabooga/text-generation-webui#5459] [Q&A]
Can anyone tell me what model and setup they've successfully used to train that model? The combinations I've tried so far have all failed. 

Thanks!

**Comment by araleza** [comment]
For training models, I usually use 4 bit GPTQ models downloaded from TheBloke. I load them with the Transformers model loader, with 'auto-devices' and 'disable_exllama' ticked, and then train them with the Training tab. Then to use the resulting LoRA, I apply them to the same model but loaded with the ExLlama_v2 loader.
